assets/sass/common/shared/o-forms/_o-form.scss

```diff
@@ -980,7 +980,7 @@ only screen and (min-resolution: 2dppx) {
 
 /* disabled state */
 .o-form-disabled,
-.o-form-saving .o-form-input > *:not(.o-form-disabled) {
+.o-form-saving .o-form-input > *:not(.o-form-disabled):not(.o-form-input-name-autoChallenge) {
```

Comment:
I would avoid changing the css globally to address any specific view.. Is there any other way ?

Reply:
I tried keeping the css in `_okta-verify.scss` but this global css overrides it still.

Reply:
`autoChallenge`  is view specific field and should not be added to global css

Reply:
make sense. I am looking at other options

Reply:
Have fixed this. please look.